finance.md

```diff
@@ -0,0 +1,69 @@
+
+### Generating timesheets
+
+1. Generate a per-member timesheet in Clockify and save (e.g. [aether_201908_bl](https://clockify.me/bookmarks/5dab94738a17084c0337690d) and [aether_201908_yj](https://clockify.me/bookmarks/5dab94828a17084c03376911))
+1. Generate a team timesheet in Clockify and save (e.g. [aether_201908](https://clockify.me/bookmarks/5dab94a0ae125662897c69b4))
+1. Export team timesheet as PDF to be sent to client, if applicable
+
+This process is done using the Clockify [Summary Report](https://clockify.me/reports/summary). Saved reports are found under [Saved Reports](https://clockify.me/reports/saved).
+
+### Generating invoices
+
+1. Generate an invoice in **Quickbooks Online** based on the pricing structure (e.g. timesheet hours and project rate)
+1. Add operational cost line item, if applicable
+1. Add GST, if applicable
+
+The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
+
+Invoices (and timesheets, if requested) are sent to clients and are payable in CAD, USD, or EUR. Invoices are due 14 days after the invoice date, after which a 5% late fee will be charged for each additional week.
+
+## Member payroll
+
+Member salaries are paid monthly and vary month-to-month depending on their billable hours of work. At the end of each month, each Member's entitled pay is calculated based on the projects and roles they participate in. After withholding taxes and other applicable deductions (e.g. CPP and EI), the net amount is electronically tranferred to the Member's bank account.
+
+Expense reimbursements, which require photos uploaded to Quickbooks Online, are paid out monthly with the salary amount as non-taxable items in the paycheque.
+
+At the end of each tax year, each Member is entitled to a vacation amount that equals 4% of their salary for the year. This amount will be added to the salary for that month.
+
+All payroll, expense reimbursements, and withheld taxes are managed using Quickbooks Online, which is linked to our main Desjardins account.
+
+### Calculating member salary
+
+Each Member's pay from billable project work will have a percent-based portion taken from it to fund administrative costs and collective programs. Collective programs may include non-billable projects, member health benefits, CPP & EI, "rainy days" reserve contributions, as well as and other collective expenses of the co-operative. The amount is calculated based on the invoiced amount before transaction costs. The exact percentage is the same across all projects during a period and can be revised as necessary by consensus.
+
+### Reporting and remitting taxes
+
+Hypha has the following accounts with the Canada Revenue Agency (CRA):
```

Comment:
Mind if we stick we their language from: https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/changes-your-business/adding-accounts-your-business-number-bn.html

I suspect the term "account" is going to be very overloaded in financial convos :)

```suggestion
Hypha has the following _program accounts_ with the Canada Revenue Agency (CRA):
```